src/config/dynamic-theme-fixes.config

```diff
@@ -17995,7 +18024,29 @@ a[title="Tinkoff"]
 
 www.tumblr.com
 
+INVERT
+.QnWzN > div
+div[role="progressbar"]
+
 CSS
+a[data-testid="google-login-button"] {
+    background-color: #0d0d0d !important;
+}
+a[data-testid="apple-login-button"] {
+    background-color: #0d0d0d !important;
+}
+a[href="/"] > div > svg {
+    filter: invert(100%) hue-rotate(180deg) contrast(200%) !important;
+}
+a[href="/explore"] > svg {
+    filter: invert(0%) hue-rotate(180deg) contrast(200%) !important;
+}
+a[aria-label="Tumblr"] > svg {
+    filter: invert(100%) hue-rotate(180deg) contrast(200%) !important;
+}
+nav.FhRnI > ul > li {
+    filter: invert(100%) hue-rotate(180deg) contrast(200%) !important;
+}
```

Comment:
Is it possible to `INVERT` these elements?

Reply:
The problem is that `INVERT` turns them light grey, but the randomly selected background is chosen/slightly altered with pure white in mind, so on occasion the objects may get lost in the background unless they are white.

For example:
![imagen](https://user-images.githubusercontent.com/71190696/156205121-7387a282-dbec-4605-adbc-0cf426481272.png)

They are still there, they are just (almost) the same color as the background.

![imagen](https://user-images.githubusercontent.com/71190696/156205095-76f9d970-b3da-4cba-b5f0-ded00b8b9f6c.png)

Although some of them simply work with `fill`, so I've changed those in the new commit. Plus two others I noticed while doing this.